source/Account.cpp

```diff
@@ -333,10 +333,19 @@ void Account::AddMortgage(int64_t principal)
 
 
 
+// Add a "fine" with a high, fixed interest rate
+void Account::AddFine(int64_t amount, int term)
+{
+	mortgages.emplace_back(amount, 0, term);
+}
+
+
+
 // Add a "fine" with a high, fixed interest rate and a short term.
-void Account::AddFine(int64_t amount)
+void Account::AddFine(int64_t amount, double interest, int term)
 {
-	mortgages.emplace_back(amount, 0, 60);
+	double creditScore = 2*(600-interest/.00001);
```

Comment:
It turns out the multiplication by int is less accurate. I end up with values that are a bit off. That is because `*100000` is not the same as `/.00001`. The later one gives you exactly the inverse of the calculation in Mortgage.cpp, which is what you need to get the correct interest rate.